index.html

```diff
@@ -7,9 +7,27 @@
   <meta name="description" content="training">
   <meta name="author" content="Wolox">
   <link rel="stylesheet"  type="text/css" href="application.css">
+  <link href="https://fonts.googleapis.com/css?family=Roboto&display=swap" rel="stylesheet">
```

Comment:
For this I suggest use and @import on applications.scss and use property font-family with universal selector *